recipes/py-rouge/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.1" %}
+
+package:
+  name: py-rouge
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/py-rouge/py-rouge-{{ version }}.tar.gz
+  sha256: b6caf2f031c45f699a9481c8962b8c33688165a3f2a22e1bfbaede8e073d6bb0
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    # - bsddb3
```

Comment:
Not sure if `py-rouge` is functional without `bsddb3`, see https://github.com/Diego999/py-rouge/issues/17 - at least the package is [importing](https://github.com/Diego999/py-rouge/blob/master/rouge/tmp/berkeley_db_to_txt.py#L1) it, albeit in a "tmp" folder

Reply:
Trying to add it for win in https://github.com/conda-forge/bsddb3-feedstock/pull/10

Reply:
At a glance it looks like it's just being used for development of the package rather than as a dependency so I think it's fine to skip